newIDE/app/src/MainFrame/EditorContainers/HomePage/CreateSection/index.js

```diff
@@ -0,0 +1,663 @@
+                  setCurrentPage={setCurrentPage}
+                  orderBy={orderBy}
+                  setGamesListOrderBy={setGamesListOrderBy}
+                  searchText={searchText}
+                  setSearchText={setSearchText}
+                />
+                {isMobile && limits && hasTooManyCloudProjects && (
+                  <MaxProjectCountAlertMessage
+                    margin="dense"
+                    limits={limits}
+                    onUpgrade={() =>
+                      openSubscriptionDialog({
+                        analyticsMetadata: {
+                          reason: 'Cloud Project limit reached',
+                        },
+                      })
+                    }
+                  />
+                )}
+                {quickCustomizationRecommendation && (
+                  <ColumnStackLayout noMargin>
+                    <Line noMargin>
+                      <Text size="block-title">
+                        {!hasAProjectOpenedOrSavedOrGameRegistered ? (
+                          <Trans>Publish your first game</Trans>
+                        ) : (
+                          <Trans>Publish a game in 1 minute</Trans>
+                        )}
+                      </Text>
+                    </Line>
+                    <QuickCustomizationGameTiles
+                      onSelectExampleShortHeader={async exampleShortHeader => {
+                        const projectIsClosed = await askToCloseProject();
+                        if (!projectIsClosed) {
+                          return;
+                        }
+
+                        const newProjectSetup: NewProjectSetup = {
+                          storageProvider: UrlStorageProvider,
+                          saveAsLocation: null,
+                          openQuickCustomizationDialog: true,
+                        };
+                        onCreateProjectFromExample(
+                          exampleShortHeader,
+                          newProjectSetup,
+                          i18n
+                        );
+                      }}
+                      quickCustomizationRecommendation={
+                        quickCustomizationRecommendation
+                      }
+                    />
+                  </ColumnStackLayout>
+                )}
+                {!hasAProjectOpenedOrSavedOrGameRegistered && (
+                  <ColumnStackLayout noMargin>
+                    <Line noMargin justifyContent="space-between">
+                      <Text size="block-title" noMargin>
+                        <Trans>Remix an existing game</Trans>
+                      </Text>
+                      <FlatButton
+                        onClick={() => setShowAllGameTemplates(true)}
+                        label={
+                          isMobile ? (
+                            <Trans>Browse</Trans>
+                          ) : (
+                            <Trans>Browse all templates</Trans>
```

Comment:
I wonder if the ResponsiveRaisedButton and ResponsiveFlatButton should have a `responsiveLabel` props. They would handle the landscape mode themselves then

Reply:
good question. though we may want to display the full label in landscape in some cases and not in others, so unsure this will simplify the reading, but a little bit!